src/components/tx/modals/TokenTransferModal/ReviewTokenTx.tsx

```diff
@@ -26,7 +27,7 @@ export const TokenTransferReview = ({
 
       <Box mt={1} fontSize={20}>
         {children}
-        {amount} {tokenInfo.symbol}
+        {formatAmount(amount)} {tokenInfo.symbol}
```

Comment:
I wouldn't use the current formatter for the reasons you stated [here](https://github.com/safe-global/web-core/pull/1265#issuecomment-1341097212). This is arguably less clear than non-formatted numbers.

Reply:
I won't merge this PR until we discuss it tomorrow

Reply:
I tend to agree it might be worse than no formatting in some (rather extreme?) cases.

Reply:
On the other hand, this is how we format numbers in other places in the app, like it or not. I don’t see why this modal has to be different than tx list, balances etc.

Reply:
This is the one place that I would argue we always display the exact precision though as assets have aribtrary values. If I were sending 100,400, I would much sooner "review" that than 100K. The same with [decimal truncation](https://github.com/5afe/safe/wiki/How-to-format-amounts).